Catappult/appcoins-wallet-android
# [WAL-103] Gas settings not stored forever

**What does this PR do?**

Adds the ability to keep the changes made on the gas settings when transferring tokens.

**Database changed?**

 No

**Where should the reviewer start?**

ConfirmationActivity.java
GasSettingsActivity.java

**How should this be manually tested?**

Try a transfer with APPC, change the gas settings, quit the transfer. Go back check if the values are the ones you've changed previously and then do a transfer and check if it's successful

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/WAL-103

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)

**Code Review Checklist**
